docs/chapter_graph/graph_operations.md

```diff
@@ -289,7 +289,9 @@ comments: true
 === "C++"
 
     ```cpp title="graph_adjacency_list.cpp"
+    [class]{Vertex}-[func]{}
```

Comment:
And I altered the markdown as in other markdown.

Reply:
is that right?

Reply:
Yes